src/pages/explore/Explore.module.css

```diff
@@ -0,0 +1,14 @@
+.paper__position {
```

Comment:
Pls use better class names & make them camelCase

Reply:
changed classname to more semantic once + make them camelcase